version.go

```diff
@@ -23,4 +23,4 @@ package tchannel
 // VersionInfo identifies the version of the TChannel library.
 // Due to lack of proper package management, this version string will
 // be maintained manually.
-const VersionInfo = "0.0.1-dev"
+const VersionInfo = "1.0.0"
```

Comment:
From the POV of someone using this library, how do they "install" `1.0.0`? Use the correlating commit hash?